### PR TITLE
New adv

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

File: server/nc-sa-2018-013.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   ],
+   "Description":"<p>A bug causing session fixation could potentially allow an attacker to obtain access to password protected shares.</p>",
+   "ActionTaken": "<p>The error has been fixed.</p>",
+   "Acknowledgment":[
+      {
+         "Name": "Anonymous hacker",
+         "Reason": "Vulnerability discovery and disclosure."
+      }
+   ],
+   "Resolution": "<p>It is recommended that all instances are upgraded to at least Nextcloud 14.0.0, Nextcloud 13.0.3 or Nextcloud 12.0.8.</p>"
```
Comment (nickvergessen) [inline server/nc-sa-2018-013.json:39]: Numbers are off, those are the affected versions.

File: server/nc-sa-2018-012.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+   ],
+   "Description":"<p>A missing access check could lead to continued access to password protected link shares when the owner had changed the password.</p>",
+   "ActionTaken": "<p>The error has been fixed.</p>",
+   "Acknowledgment":[
+      {
+         "Name": "Rudra Pratap Singh",
+         "Reason": "Vulnerability discovery and disclosure."
+      }
+   ],
+   "Resolution": "<p>It is recommended that all instances are upgraded to Nextcloud 14.0.0.</p>"
```
Comment (nickvergessen) [inline server/nc-sa-2018-012.json:28]: this is the affected version

File: server/nc-sa-2018-013.json (hunk 1 above)
Comment (rullzer) [reply server/nc-sa-2018-013.json:39]: Nope affected is `<`

File: server/nc-sa-2018-012.json (hunk 2 above)
Comment (rullzer) [reply server/nc-sa-2018-012.json:28]: Check the operator

**Comment by MorrisJobke** [comment]
Should we wait for the CVE assignment?

**Comment by rullzer** [comment]
> Should we wait for the CVE assignment?

No can take a while. Also MITRE wants public references. H1 is disclosed already but more is better. We can just update them later.

**Review by MorrisJobke — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Let me publish them on the website then

**Pull request merged**